bot/app/database/db_objects.py

```diff
@@ -77,6 +81,24 @@ def is_event(self):
         conn.close()
 
         return search is not None
+    
+    def get_max_event(self):
+
+        conn = sqlite3.connect(DB_NAME)
+        curs = conn.cursor()
+
+        try:
+            curs.execute("""select max(eventID) from Events""")
+            event_id = curs.fetchone()
+            if event_id is not None:
+                event_id + 1
+        except:
+            event_id = 000000 
+
+        curs.close()
+        conn.close()  
+
+        return event_id
```

Comment:
This is unnecessary with [AUTOINCREMENT](https://www.sqlite.org/faq.html#q1), which I believe you are already using.

Reply:
fixed